manifests/4.3/cluster-logging.v4.3.0.clusterserviceversion.yaml

```diff
@@ -57,7 +57,31 @@ metadata:
                     "type": "fluentd",
                     "fluentd": {}
                   }
-                }
+                },
+                "forwarding" {
+                  "outputs": [
+                    {
+                      "name": "clo-default-output-es",
+                      "type": "elasticsearch",
+                      "endpoint": "elasticsearch.openshift-logging.svc:9200",
+                      "secret" {
+                        "name", "elasticsearch"
+                      }
+                    }
+                  ],
+                  "pipelines": [
```

Comment:
are we sold on "pipelines" ? in this context "sources" may make more sense...

Reply:
> are we sold on "pipelines" ? in this context "sources" may make more sense...

We discussed this previously (though @sichvoge was not found of the term) and agreed on `pipelines` because we are wiring a source to a destination.  I don't think sources accurately conveys what we are abstracting here but given this will endup being a TP api we can revisit it before GA

> 
> reading further down in the CRD description i see that they're meant to connect sources and outputs.. i think this doesn't read as obvious to me because the `type` specifies the source type even though this is a `pipeline` but we also have our `outputRefs`... can we s/type/inputType/ ?

Same statement regarding TP vs GA though I think `inputType` makes sense in this context given we are trying to describe how a given source is being sent to a destination.  I'll update it accordinly